python/rpdk/java/templates/init/guided_aws/StubUpdateHandler.java

```diff
@@ -26,126 +25,88 @@
 
         this.logger = logger;
 
-        final ResourceModel model = request.getDesiredResourceState();
-
         // TODO: Adjust Progress Chain according to your implementation
         // https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/blob/master/src/main/java/software/amazon/cloudformation/proxy/CallChain.java
 
-        return ProgressEvent.progress(model, callbackContext)
+        return ProgressEvent.progress(request.getDesiredResourceState(), callbackContext)
             // STEP 1 [first update/stabilize progress chain - required for resource update]
             .then(progress ->
                 // STEP 1.0 [initialize a proxy context]
-                proxy.initiate("{{ call_graph }}::{{ operation }}", proxyClient, model, callbackContext)
+                // Implement client invocation of the update request through the proxyClient, which is already initialised with
+                // caller credentials, correct region and retry settings
+                proxy.initiate("{{ call_graph }}::{{ operation }}::first", proxyClient, progress.getResourceModel(), progress.getCallbackContext())
```

Comment:
::first
```suggestion
                proxy.initiate("{{ call_graph }}::{{ operation }}::sendUpdateRequest", proxyClient, progress.getResourceModel(), progress.getCallbackContext())
```

Reply:
I meant that it's the first update in case of multiple update calls required